syntax.qmd

```diff
@@ -308,7 +308,7 @@ hierarchy easy to see:
 
 * `}` should be the first character on the line.
 
-It is occassionally useful to have an empty curly braces block, in which case it should be written `{}`.
+It is occassionally useful to have empty braced expressions, in which case it should be written `{}`.
```

Comment:
```suggestion
It is occassionally useful to have an empty braced expression, in which case it should be written `{}`. For example, if you need a function that doesn't do anything, you can write `function(...) {}`.
```

Reply:
I think this conflicts a bit with the native lambda advice, I would use "you can write `\(...) {}`"

https://style.tidyverse.org/functions.html#anonymous-functions

Reply:
This example is clearer with `function` than `\` though. The function that doesn't do anything isn't necessarily anonymous, for instance a function stub at top level of a package (to be overwritten in `.onLoad`).